tests/test_filter_count.py

```diff
@@ -930,9 +932,21 @@ def test_batched_label_issues():
         batch_size=len(data["labels"]) + 100,
         **extra_args,
     )
+    assert not np.array_equal(f5, f2)
     assert np.all(f7 == f5)
     assert np.all(f6 == f5)
     assert np.abs(len(f5) - n1) < 2
+    # Test batches loaded from file:
+    labels_file = path.join(mkdtemp(), "labels.npy")
+    pred_probs_file = path.join(mkdtemp(), "pred_probs.npy")
+    np.save(labels_file, data["labels"])
+    np.save(pred_probs_file, data["pred_probs"])
+    f8 = find_label_issues_batched(
+        labels_file=labels_file,
+        pred_probs_file=pred_probs_file,
+        batch_size=int(len(data["labels"]) / 4.0),
+    )
+    assert np.all(f8 == f3)
```

Comment:
Doesn't f8 batch size directly compare to f2 not f3?

Reply:
I purposefully want to compare it against the run with a different batch_size. The batch_size should never affect the results